etcdserver/etcdserverpb/rpc.proto

```diff
@@ -78,6 +78,21 @@ message RangeRequest {
   // if the revision has been compacted, ErrCompaction will be returned in
   // response.
   int64 revision = 4;
+
+  enum SortOrder {
+	NONE = 0; // default, no sorting
+	ASCEND = 1; // lowest key first
```

Comment:
probably lowest value first (we can compare key,value of key, version, create, mod etc. not just key)